app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt

```diff
@@ -254,9 +254,9 @@ open class GameActivity : AppCompatActivity(), GameView, Timer.Listener {
 
     protected open fun initVoiceChat(gameUid: Long) {
 
-        val map = mapIdToName.mapKeys { it.hashCode().absoluteValue }
+        val map = mapIdToName.mapKeys { it.key.hashCode().absoluteValue }
         if (!this::voiceChannel.isInitialized) voiceChannel = VoiceIpEngineDecorator(this, MyIRtcEngineEventHandler(this, map))
-        val userId = user.name.hashCode().absoluteValue
+        val userId = user.id.hashCode().absoluteValue
```

Comment:
Yes, in fact I never wanted to put user.name instead of user.id it was a typeface. To calculate the agora id the goal is to use the player id but as the players id can contain letters and agora uses an int as id, I hash the player id to get the agora id.

Reply:
Nice, good idea!